src/extension/executors/aws.ts

```diff
@@ -41,67 +41,14 @@ export const aws: IKernelExecutor = async (executor) => {
       cellId,
     })
 
-    // todo(sebastian): move down into kernel?
     switch (programOptions.exec?.type) {
       case 'script':
-        {
-          programOptions.exec.script = 'echo $AWS_PROFILE'
-        }
+        programOptions.exec.script = 'echo $AWS_PROFILE'
         break
     }
 
-    const program = await runner.createProgramSession(programOptions)
```

Comment:
I took advantage to avoid code duplication.

Reply:
Good idea to DRY. This will do for the simple cases.